roles/wildfly_systemd/tasks/yml_config_version_prereq.yml

```diff
@@ -7,10 +7,25 @@
   when:
     - eap_enable is not defined or (not eap_enable | bool)
 
+- name: "Check latest applied patch checksum file"
+  become: yes
```

Comment:
Let's have a variable for the become:, this will save us some trouble, down the road, when people use the collection to deploy the server on system they don't have root access.